src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.props

```diff
@@ -50,12 +50,6 @@
     </ItemGroup>
   </Target>
 
-  <!-- Mobile uses a different hosting model, so we don't include the .NET host components. -->
```

Comment:
nit, if you need to touch the PR again: preserve this comment in liveBuilds.targets as I think it is useful